strawberry/experimental/pydantic/exceptions.py

```diff
@@ -5,7 +5,11 @@
 
 class MissingFieldsListError(Exception):
     def __init__(self, type: Type[BaseModel]):
-        message = f"List of fields to copy from {type} is empty"
+        message = (
+            f"List of fields to copy from {type} is empty. Either pass a "
+            f"`fields` list, set `all_fields` to True, or add fields with the "
```

Comment:
I think we can just recommend the `auto` api here since that's what we're changing the docs to use.